Mathlib/Analysis/Calculus/Deriv/Basic.lean

```diff
@@ -69,8 +69,9 @@ and they more frequently lead to the desired result.
 
 We set up the simplifier so that it can compute the derivative of simple functions. For instance,
 ```lean
-example (x : ℝ) : deriv (fun x ↦ cos (sin x) * exp x) x = (cos(sin(x))-sin(sin(x))*cos(x))*exp(x) :=
-by { simp, ring }
+example (x : ℝ) :
+    deriv (fun x ↦ cos (sin x) * exp x) x = (cos(sin(x))-sin(sin(x))*cos(x))*exp(x) := by
```

Comment:
I'm not even sure the over-parenthesized syntax works in lean 4

Reply:
Trying on the playground: [This syntax parses](https://live.lean-lang.org/#code=import%20Mathlib%0A%0Aopen%20Real%0A%0Aexample%20(x%20%3A%20%E2%84%9D)%20%3A%20deriv%20(fun%20x%20%E2%86%A6%20cos%20(sin%20x)%20*%20exp%20x)%20x%20%3D%20(cos%20(sin%20x)%20-%20sin%20(sin%20x)%20*%20cos%20x)%20*%20exp%20x%20%3A%3D%20by%0A%20%20simp%3B%20ring%0A%0Aexample%20(x%20%3A%20%E2%84%9D)%20%3A%20deriv%20(fun%20x%20%E2%86%A6%20cos%20(sin%20x)%20*%20exp%20x)%20x%20%3D%0A%20%20%20%20(cos%20(sin%20x)%20-%20sin%20(sin%20x)%20*%20cos%20x)%20*%20exp%20x%20%3A%3D%20by%0A%20%20simp%3B%20ring), without the spaces it does not. It seems the example fails, though...